### PR TITLE
[ README ] fix typo & add link for cc APIs

### PR DESCRIPTION
**Pull request by jijoongmoon**
fix typo & add link for cc APIs

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ NNtrainer is an Open Source Project. The aim of the NNtrainer is to develop a So
 
 Even though NNTrainer can be used to train sub-models, it requires implementation of additional functionalities to train models obtained from other machine learning and deep learning libraries. In the current version, various machine learning algorithms such as k-Nearest Neighbor (k-NN), Neural Networks, Logistic Regression and Reinforcement Learning algorithms are implemented. We also provide examples for various tasks such as transfer learning of models. In some of these examples, deep learning models such as Mobilenet V2 trained with Tensorflow-lite, are used as feature extractors. All of these were tested on Galaxy S8 with Android and PC (Ubuntu 16.04).
 
+[ NNTrainer: Personalize neural networks on devices! ](https://www.youtube.com/watch?v=HKKowY78P1A), Samsung Developer Conference 2021
```
Comment (kparichay) [inline README.md:21]: Let's add this as well: https://www.youtube.com/embed/Jy_auavraKg?start=4035&end=4080

**Review by lhs8928 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jijoongmoon) [reply README.md:21]: Great. Thanks.

**Review by zhoonit — APPROVED** [review]
LGTM :)

**Review by kparichay — APPROVED** [review]
LGTM

**Pull request merged**